spec/features/dashboards_spec.rb

```diff
@@ -29,6 +29,24 @@
     end
   end
 
+  describe "static dashboards" do
+    scenario "creating a static dashboard" do
+      dashboard = FactoryGirl.create(:dashboard)
+      visit dashboard_slug_path dashboard.slug
+      expect {
+        find('.glyphicon-link').click
```

Comment:
thoughts on this? it's just an icon so i'm targeting a class, but using a generic class is a bad idea (even if right now it is unique), but i'm not sure how i feel about adding a testing-specific class.